blocks/carousel/carousel.css

```diff
@@ -137,19 +127,41 @@
   left: calc(50% - 2px);
 }
 
-@media (width >= 600px) {
-  .carousel .carousel-navigation-buttons {
-    left: -110px;
-    right: -110px;
-  }
-
+@media (width >= 300px) {
```

Comment:
You can remove this media query, just set these as the default styles. There are no modern phones less than 300px wide.